### PR TITLE
Bumped activity exemptions with checkbox regression fix

### PR DESCRIPTION
**Pull request by BraydonKains**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4917,7 +4917,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#b2f0a3833cb8406386d055de4dd836ab9d269978",
+      "version": "github:Brightspace/d2l-rubric#f738ed5186ada6f06b1b89df006c927746db7421",
```
Comment (BraydonKains) [inline package-lock.json]: FYI @mdgbayly

File: package-lock.json [hunk 2]
```diff
@@ -4825,7 +4825,7 @@
       }
     },
     "d2l-outcomes-user-progress": {
-      "version": "github:Brightspace/d2l-outcomes-user-progress#a1e4acf47a0cb64dad87d3e4e6bd153de883b61b",
+      "version": "github:Brightspace/d2l-outcomes-user-progress#24e43f4e1f1236b298e4b2eb06d439f695299a34",
```
Comment (BraydonKains) [inline package-lock.json]: I'm sorry I don't know who owns this

File: package-lock.json (hunk 1 above)
Comment (jwong-d2l) [inline package-lock.json]: @sarahboyd fyi

File: package-lock.json (hunk 2 above)
Comment (jwong-d2l) [inline package-lock.json]: @rgoessl fyi

File: package-lock.json [hunk 3]
```diff
@@ -26723,7 +26723,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#6bf49464909e0b78d3cd8762ff17d42305287867",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#83e9afd5b0dc82f262b45d5262bb2b9c23df6be6",
```
Comment (jwong-d2l) [inline package-lock.json]: @svanherk fyi

**Pull request merged**